src/imap.rs

```diff
@@ -2022,6 +2070,20 @@ async fn mark_seen_by_uid(
     }
 }
 
+pub(crate) async fn markseen_on_imap(context: &Context, message_id: &str) -> Result<()> {
```

Comment:
i'd rename this to `markseen_on_imap_table` or so to make it obvious this is not doing IMAP-operations.

Reply:
Renamed and added a doc comment